.travis.yml

```diff
@@ -1,2 +1,8 @@
 script:
-- echo "hello world"
+- docker login --username devopscorndel --password $dockerhubpass
+- docker build --target production --tag devopscorndel/travisimage:latest .
+- docker push devopscorndel/travisimage:latest
+- docker login --username=rusab36@gmail.com --password=$herokutoken registry.heroku.com
+- docker pull devopscorndel/travisimage:latest
```

Comment:
We don't need to pull this image here because we just pushed it

Reply:
done